docs/src/pages/docs/Connecting to Databases/postgres.mdx

```diff
@@ -25,13 +25,13 @@ Postgres sample connection parameters:
 The connection string looks like:
 
 ```
-postgresql+psycopg2://{username}:{password}@{host}:{port}/{database}
+postgresql://{username}:{password}@{host}:{port}/{database}
 ```
 
-Extra parameters:
+You can require SSL by adding `?sslmode=require` at the end:
```

Comment:
Do you think it would make sense to add a reference to: https://www.postgresql.org/docs/9.1/libpq-ssl.html
section `Table 31-1. SSL Mode Descriptions` it has all sslmode's supported by postgres

Reply:
Good point, will do!